lib/publiccloud/instance.pm

```diff
@@ -85,6 +86,43 @@ sub run_ssh_command {
     }
 }
 
+=head2 ssh_script_retry
+
+    ssh_script_retry(command[, retry => 3][, delay => 10][, timeout => 90][, ssh_opts =>'..'][, username => 'XXX'][, no_quote => 0]);
+
+Run a C<command> via ssh in the given PC instance until it succeeds or
+the given number of retries is exhausted and an exception is thrown.
+Timeout can be set by C<timeout> or 90 sec by default.
+By default, the command is passed in single quotes to SSH.
+To avoid quoting use C<<no_quote=>1>>.
+With C<<ssh_opts=>'...'>> you can overwrite all default ops which are in
+C<<$instance->ssh_opts>>.
+Use argument C<username> to specify a different username then
+C<<$instance->username()>>.
+=cut
+sub ssh_script_retry {
```

Comment:
```suggestion
sub retry_ssh_command {
```

Reply:
While your suggestion looks nicer per se, I would argue, that `ssh_script_retry` is closer to the already established `script_retry` API call and we stay somehow consistent

Reply:
I also want to be consistent. And as we already have `run_ssh_command` in the same module, I'd do `retry_ssh_command`.

Reply:
I guess consistency within the module trumps over consistency with the testapi. That's ok with me :+1: